### PR TITLE
Feat : Added Homepage

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

Can you please review my homepage.

Link - https://water-commerce.netlify.app/

I'm trying to follow the figma template given to us in LMS.

Thanks

### CODE REVIEW

**Review by sahiba0915 — COMMENTED** [review]
overall looks good to me, work on responsiveness side

File: index.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "https://water-style-ui.netlify.app/main.css">
+    <link rel = "stylesheet" href = "styles.css">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <title>Document</title>
```
Comment (darshandamre) [inline index.html]: give a title

File: index.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+       <div class = "flex gap-xl flex-center grow-1" >
+           <button class="btn btn-secondary">Login</button>
+           <i class="material-icons" >favorite_border</i>
+           <i class="material-icons" >shopping_cart</i>
+       </div>
+   </nav>
+
+   <div class = "m5-top"></div>
+
+   <section class = "preview container center-block flex space-between">
```
Comment (darshandamre) [inline index.html]: no need of empty div, you can directly give margin to the section

File: index.html [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+          />
+          <span class="badge-icon">New</span>
+          <span class="overlay-text">New Stock</span>
+        </div>
+      </div>
+    </section>
+
+    <div class = "m5-top"></div>
+
+    <section class = "hero container center-block">
```
Comment (darshandamre) [inline index.html]: same here

**Review by darshandamre — COMMENTED** [review]
i have some suggestions

File: index.html (hunk 2 above)
Comment (shlok30) [reply index.html]: Understood. thanks

**Pull request merged**